docs/go-run-your-first-app.md

```diff
@@ -65,6 +70,8 @@ There are two ways to start a Workflow with Temporal, either via the SDK or via
 <!--SNIPSTART money-transfer-project-template-go-start-workflow-->
 <!--SNIPEND-->
 
+### Running the Workflow
+
```

Comment:
the reason i make this a small header is to make it visually stand out since the other code samples are for reading, whereas here an action from the reader is actually expected. 

is there a better way to do this?